Retail/src/retail/Retail.java

```diff
@@ -0,0 +1,107 @@
+package retail;
+import java.util.Scanner;
+import retailPackage.RetailItem;
+
+public class Retail {
+
+    /**
+     * Main program that calls the retail class
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        
+        //variable init
+        String discription;
```

Comment:
Thank you! I have just updated that typo. That "replace" option that Paul showed us in class definitely came in use for me.